h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -832,12 +851,58 @@ private void fitIRLSM(Solver s) {
             Log.info(LogMsg("computed in " + (t2 - t1) + "+" + (t3 - t2) + "+" + (t4 - t3) + "=" + (t4 - t1) + "ms, step = " + ls.step() + ((_lslvr != null) ? ", l1solver " + _lslvr : "")));
           } else
             Log.info(LogMsg("computed in " + (t2 - t1) + "+" + (t3 - t2) + "=" + (t3 - t1) + "ms, step = " + 1 + ((_lslvr != null) ? ", l1solver " + _lslvr : "")));
+
+/*          if (_parms._optimize_theta && (iterCnt > 1)&& _parms._family.equals(Family.negativebinomial)) {
```

Comment:
please remove all the commented out code

Reply:
Done